api/envoy/config/trace/v3/trace.proto

```diff
@@ -10,6 +10,7 @@ import public "envoy/config/trace/v3/http_tracer.proto";
 import public "envoy/config/trace/v3/lightstep.proto";
 import public "envoy/config/trace/v3/opencensus.proto";
 import public "envoy/config/trace/v3/service.proto";
+import public "envoy/config/trace/v3/skywalking.proto";
```

Comment:
The `import public` around this proto is just for backward compatibility during re-structuring proto files, new proto doesn't have to be here.

Reply:
How about `backend_token`. Because how the authentication active is based on skywalking backend setting.